integration-tests/ws-trust/src/test/java/io/quarkiverse/cxf/it/ws/trust/CxfWsTrustTest.java

```diff
@@ -76,28 +80,28 @@ void wsdl() {
 
     @Test
     public void programmaticSts() throws Exception {
-        Bus bus = BusFactory.newInstance().createBus();
-        try {
-            BusFactory.setThreadDefaultBus(bus);
-
-            final QName serviceName = new QName("https://quarkiverse.github.io/quarkiverse-docs/quarkus-cxf/test/ws-trust",
-                    "TrustHelloService");
-            final URL wsdlURL = new URL(io.quarkiverse.cxf.test.QuarkusCxfClientTestUtil.getServerUrl()
-                    + "/services/jaxws-samples-wsse-policy-trust/TrustHelloService?wsdl");
-            Service service = Service.create(wsdlURL, serviceName);
-            TrustHelloService proxy = (TrustHelloService) service.getPort(TrustHelloService.class);
-
-            final QName stsServiceName = new QName("http://docs.oasis-open.org/ws-sx/ws-trust/200512/", "SecurityTokenService");
-            final QName stsPortName = new QName("http://docs.oasis-open.org/ws-sx/ws-trust/200512/", "UT_Port");
-
-            String stsURL = QuarkusCxfClientTestUtil.getServerUrl()
-                    + "/services/jaxws-samples-wsse-policy-trust-sts/SecurityTokenService?wsdl";
-            setupWsseAndSTSClient(proxy, bus, stsURL, stsServiceName, stsPortName);
-
-            Assertions.assertThat(proxy.sayHello()).isEqualTo("WS-Trust Hello World!");
-        } finally {
-            bus.shutdown(true);
-        }
+
+        BusFactory.setThreadDefaultBus(bus);
+
+        //programmatic registration of customizedAlgorithmSuite
```

Comment:
@ffang I had to register custom algorithm in this test. I suppose that in case of programmatic STS programatic registration makes sense, what do you think?